### PR TITLE
Feature signup

### PR DESCRIPTION
**Pull request by meenakshig-cuelogic**
Have added landing page in landingpage.html
Updated signup.html
Have added email logic in views

### CODE REVIEW

File: AnalysisReporting/.cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "AnalysisReporting/test_setting.py": true
+}
```
Comment (ananthu-nair707) [inline AnalysisReporting/.cache/v/cache/lastfailed]: Please remove this file

File: AnalysisReporting/.coverage [hunk 2]
```diff
@@ -0,0 +1 @@
...ages/django/forms/__init__.py":[3,5,6,7,8,9,10,11],"/home/meenakshig/.local/lib/python2.7/site-packages/django/core/checks/messages.py":[2,4,7,8,9,10,11,14,15,17,18,19,20,21,22,23,25,32,35,50,54,57,62,63,67,68,72,73,74,77,78,79,82,83],"/home/meenakshig/.local/lib/python2.7/site-packages/django/core/cache/__init__.py":[128,14,15,17,18,19,22,25,26,29,32,58,63,64,65,67,84,88,91,97,98,101,104,107,110,113,117,120],"/home/meenakshig/.local/lib/python2.7/site-packages/django/template/__init__.py":[66,69,40,44,45,47,49,55,56,57,60],"/home/meenakshig/.local/lib/python2.7/site-packages/django/views/generic/edit.py":[1,2,3,4,5,6,257,11,268,14,143,16,17,18,19,21,279,27,284,158,287,33,291,292,134,166,39,169,170,47,176,189,63,193,196,199,202,75,205,81,210,211,215,88,228,271,220,224,97,100,101,103,233,234,225,238,243,247,248,251,254,255],"/home/meenakshig/.local/lib/python2.7/site-packages/django/db/migrations/exceptions.py":[1,3,4,7,10,11,14,17,18,21,24,25,28,31,32,35,38,39,42,45,46,49,50,53,55,60,63,67,68,71,72],"/home/meenakshig/.local/lib/python2.7/site-packages/django/db/models/fields/related_lookups.py":[128,1,132,133,7,8,137,10,140,141,14,144,145,18,148,149,23,129,43,44,136,59,97,98,114]}}
```
Comment (ananthu-nair707) [inline AnalysisReporting/.coverage]: Remove this file

File: AnalysisReporting/AnalysisReporting/settings.py [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+EMAIL_HOST_PASSWORD='Cuelogic'
+Email_PORT=587
+
+
+
+
+
+
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
```
Comment (ananthu-nair707) [inline AnalysisReporting/AnalysisReporting/settings.py]: Remove unnecessary spaces here.

File: AnalysisReporting/AnalysisReporting/settings.py [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+USE_TZ = True
+
+
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/dev/howto/static-files/
+
+STATIC_URL = '/static/'
+STATICFILES_DIRS=[os.path.join(BASE_DIR,"static")]
+LOGIN_REDIRECT_URL = 'home'
```
Comment (ananthu-nair707) [inline AnalysisReporting/AnalysisReporting/settings.py]: Please follow EOL rule, as mentioned in PEP8

File: AnalysisReporting/analysisreport/admin.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.contrib import admin
+
```
Comment (ananthu-nair707) [inline AnalysisReporting/analysisreport/admin.py:5]: Do we need this file? If yes, please follow EOL rule.

File: AnalysisReporting/analysisreport/forms.py [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+from django.http import Http404
+from django.shortcuts import render,redirect
+from django.contrib.auth import authenticate,login
+from django.views.generic import View
+from django.forms import ModelForm
+from django import forms
+
+ 
+
+
```
Comment (ananthu-nair707) [inline AnalysisReporting/analysisreport/forms.py]: Remove unnecessary spaces

File: AnalysisReporting/analysisreport/models.py [hunk 7]
```diff
@@ -0,0 +1,13 @@
+
+from django.contrib.auth.models import User
+from django.db import models
+
+# Create your models here.
+
+
```
Comment (ananthu-nair707) [inline AnalysisReporting/analysisreport/models.py]: Please follow this for spacing between code blocks 
https://www.python.org/dev/peps/pep-0008/#blank-lines

File: AnalysisReporting/analysisreport/models.py [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+# Create your models here.
+
+ 
+class emailverify(models.Model):
+    username = models.ForeignKey(User, on_delete=models.CASCADE)
+    hashkey = models.CharField(max_length=50)
+    
+    
+
```
Comment (ananthu-nair707) [inline AnalysisReporting/analysisreport/models.py]: Remove blank lines

File: AnalysisReporting/analysisreport/static/analysisreport/css/landingpage.css [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+    {
+        text-align: center;
+        padding: 30px;
+        font-size: 50px;
+        color: white;
+        line-height: 1.3;
+        text-rendering: optimizeLegibility;
+       
+    }
+
```
Comment (ananthu-nair707) [inline AnalysisReporting/analysisreport/static/analysisreport/css/landingpage.css]: Remove blank lines.

**Review by ananthu-nair707 — CHANGES_REQUESTED** [review]
Please make necessary changes for python code as mentioned in comments.

File: AnalysisReporting/analysisreport/static/analysisreport/css/landingpage.css [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+    {
+       background-color: black;
+       opacity: 0.5;
+       position: absolute;
+        height: 100%;
+        width: 100%;
+        top: 0;
+        left: 0;
+    }
+.jumbotron-content
```
Comment (nileshm-cuelogic) [inline AnalysisReporting/analysisreport/static/analysisreport/css/landingpage.css:31]: Remove unnecessary spacing, and do proper formatting to the code.

File: AnalysisReporting/analysisreport/static/analysisreport/css/landingpage.css [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+    {
+
+    }
+    .container-fluid
+    {
+        background-color: #3396ff;
+    }
+    
+
+    .font_l
```
Comment (nileshm-cuelogic) [inline AnalysisReporting/analysisreport/static/analysisreport/css/landingpage.css:68]: Class names should be consistent, instead of using _ in class name use  -

File: AnalysisReporting/analysisreport/static/analysisreport/js/validations.js [hunk 12]
```diff
@@ -0,0 +1,168 @@
+// $(document).ready(function() {
```
Comment (nileshm-cuelogic) [inline AnalysisReporting/analysisreport/static/analysisreport/js/validations.js]: Do not commit unwanted commented code.

File: AnalysisReporting/analysisreport/templates/analysisreport/landingpage.html [hunk 13]
```diff
@@ -0,0 +1,85 @@
+{% load static%}
+<!DOCTYPE html>
+<html>
+<head>
+	<title>login</title>
+
+	<meta charset="utf-8">
+    <title> Signup </title>
```
Comment (nileshm-cuelogic) [inline AnalysisReporting/analysisreport/templates/analysisreport/landingpage.html]: There are two title tags in the header section, keep only one.  
Wrong title given to landing page, change it.

File: AnalysisReporting/analysisreport/templates/analysisreport/landingpage.html [hunk 14]
```diff
@@ -0,0 +1,85 @@
...
+    
+     <script src="{% static 'analysisreport/bootstrap-3.3.7-dist/js/bootstrap.min.js' %}"></script>
+     <script src="{% static 'analysisreport/js/jquery.validate.js' %}"></script>
+
+ <script src="{% static 'analysisreport/js/validations.js' %}"></script>
+
+
+<script src="analysisreport/js/Cookie/jquery.cookie.js"></script>
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
```
Comment (nileshm-cuelogic) [inline AnalysisReporting/analysisreport/templates/analysisreport/landingpage.html]: jquery.min.js is included two times. keep only one script.

File: AnalysisReporting/analysisreport/templates/analysisreport/landingpage.html [hunk 15]
```diff
@@ -0,0 +1,85 @@
...
+<link href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700" rel="stylesheet">
+    
+<link rel="stylesheet" type="text/css" href="{% static 'analysisreport/css/landingpage.css' %}" />
+ 
+   
+ 
+
+</head>
+        <body>
```
Comment (nileshm-cuelogic) [inline AnalysisReporting/analysisreport/templates/analysisreport/landingpage.html]: Do proper code formatting.

File: AnalysisReporting/analysisreport/templates/analysisreport/loginpage.html [hunk 16]
```diff
@@ -0,0 +1,113 @@
...
+  <h1 style="height: 20px; margin-top: -2px;  ">Analysis & Reporting</h1>      
+  </header>
+
+
+
+ <div class="container" style="text-align:center;background-color: lightgrey; "> 
+
+ <h3 style="text-align:center;"> Login </h3>
+ 
+<br>
```
Comment (nileshm-cuelogic) [inline AnalysisReporting/analysisreport/templates/analysisreport/loginpage.html]: do not use <br> tag spacing and margin purpos. use padding and margin properties from CSS.

File: AnalysisReporting/analysisreport/templates/analysisreport/loginpage.html [hunk 17]
```diff
@@ -0,0 +1,113 @@
...
+            </div>
+            <div class="col-sm-3 col-sm-pull-2">
+
+  
+
+                <p>New User? <a href="register">create an account </a></p>
+             </div>
+        </div>
+
+  </DIV>
```
Comment (nileshm-cuelogic) [inline AnalysisReporting/analysisreport/templates/analysisreport/loginpage.html]: tag name should be in lower case only
